anthy/README.md

```diff
@@ -1,15 +1,27 @@
 # Anthy
 
-## How to build anthy in sb2
-
-1. `tar xf anthy-9100h.tar.gz && cd anthy-9100h/`
-2. `sb2 -t SailfishOS-* -m sdk-install -R zypper in gcc make automake libtool`
-3. `mb2 -t SailfishOS-* -s ../anthy.spec build`
-4. `sb2 -t SailfishOS-* -m sdk-build -R`
-5. `export LD_LIBRARY_PATH="$(pwd)/src-main/.libs:$(pwd)/src-worddic/.libs:${LD_LIBRARY_PATH}"`
-6. `make`
-7. `exit`
-8. `mb2 -t SailfishOS-* -s ../anthy.spec build`
+## Build Anthy with Sailfish SDK
+
+```bash
+# configure SDK
+PATH=$PATH:~/SailfishOS/bin/
+
+export OS_VERSION=4.1.0.24
+export ARCHITECTURE=aarch64
```

Comment:
This wouldn't work on a 32-bit arm architecture, right? Should probably add separate build instructions for those as well.

Reply:
Of course. these variables (PATH, OS_VERSION, ARCHITECTURE) have to be initialized regarding your installation and target platform. I added comments to them...

Reply:
> I added comments to them...

Looks good